edx_django_utils/monitoring/internal/code_owner/middleware.py

```diff
@@ -0,0 +1,147 @@
+"""
+Middleware for code_owner custom attribute
```

Comment:
**Reviewer Note:** Outside of minor updates (e.g. import updates, minor doc changes involving imports, etc.), the logic of moved files should be mostly unchanged.
Please mark as resolved once you've read this.